account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
-		goa.LogError(ctx, "error adding Identity", "error", err.Error())
+		log.LogError(ctx, map[string]interface{}{
```

Comment:
> I'm not sure but I think it is also considered bad practice to log the error here if we don't handle it?

@kwk `goa.LogError` does log the error. Why did this error is there ? I think it is a bit soon for me to remove logs which, IMHO, add some value.

The intention here is replace the goa logging by ours. I don't blindly replace the old log msgs. I added additional structured information that could help us to trace the problem in the future. This will also helps us to store the information in a JSON friendly format.